src/java_bytecode/java_bytecode_instrument.cpp

```diff
@@ -82,6 +83,21 @@ const std::vector<std::string> exception_needed_classes = {
   "java.lang.NegativeArraySizeException",
   "java.lang.NullPointerException"};
 
+/// Get a copy of a source location with additional assertion information
+/// \param loc: original source location
+/// \param comment: human-readable assertion description
+/// \param property_class: assertion property class
+static source_locationt with_assertion_description(
```

Comment:
Possibly useful, but please make this more generally available. On quick inspection almost every place that `git grep -n set_comment` highlights could benefit from this.

Reply:
I don't think that makes the code more readable.